assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -187,11 +183,18 @@ function Title({
     });
   }, [panelTitleReadable, isCollapsed, toggle]);
 
+  const titleLabel = String(children) + __(' panel', 'web-stories');
```

Comment:
We should avoid string concatenation for localized strings. 

> Use format strings instead of string concatenation

https://codex.wordpress.org/I18n_for_WordPress_Developers

More info here: http://www.gnu.org/software/gettext/manual/html_node/Preparing-Strings.html#Preparing-Strings

Reply:
Yeah if anything it would need to be like this:

```js
/* translators: %s: Panel title. */
const titleLabel = sprintf('%s panel', panelTitle);
```

But note that you can't easily do this here as `children` could be anything, even a React component! You can't just do `String()` on that.

There would need to be a new prop for passing a plaintext title.

Reply:
That being said, does it really need to be announced as "xyz panel"? How is it being announced today?

Reply:
Thanks for the suggestion, I created a `title` props at the parent, and passed it as a string.